talk/owt/sdk/base/stream.cc

```diff
@@ -439,6 +451,21 @@ std::shared_ptr<LocalStream> LocalStream::Create(
 }
 #endif
 
+#ifdef OWT_ENABLE_QUIC
+LocalStream::LocalStream(std::shared_ptr<QuicStream> writer) {
```

Comment:
It's not a `writer`. `stream` or `quic_stream` would be better.

Reply:
done